doc/src/modules/solvers/solveset.rst

```diff
@@ -6,62 +6,83 @@ Solveset
 .. module:: sympy.solvers.solveset
 
 This is the official documentation of the ``solveset`` module in solvers.
-It contains the frequently asked questions about our new module to solve
-equations.
 
 .. note::
 
    For a beginner-friendly guide focused on solving common types of equations,
    refer to :ref:`solving-guide`.
 
-What's wrong with solve():
---------------------------
+How solveset() is different from solve()
+----------------------------------------
+
+SymPy already has a pretty powerful ``solve`` function. But it has different
+goals. For example:
+
+* It has a generalized output to handle various types of solutions.
+  It needs to return a lot of types of solutions consistently. It can deal with
+  univariate as well as multivariate equations as well as various other sorts
```

Comment:
This clumsy wording - various other sorts of variability (ick!) - will be revised in the next draft.